### DISCUSSION TITLE
How to distinguish rail ferry/shuttle train from normal rails

### DISCUSSION THREAD

**Discussion by Azbesciak** [OvertureMaps/data#300] [Discussion]
Hi, I am looking for a way how to differentiate shuttle train from normal rails. My only idea is shuttle trains (like in canal La Manche) may have the name, but I also saw cases for rails, which are not shuttle trains... which also have name.
https://explore.overturemaps.org/#14.95/50.99471/1.56269

**Comment by brad-richardson** [comment]
Hi @Azbesciak, unfortunately there is currently no way to distinguish rail types as we've not yet defined a classification schema. It's a current area of interest for us, are there any other specific railway types you're interested in? We're looking at the most popular tag values for OSM's [`railway` tag](https://taginfo.openstreetmap.org/keys/railway#values) and [`railway:traffic_mode`](https://taginfo.openstreetmap.org/keys/railway%3Atraffic_mode#values).

**Reply by Azbesciak** [reply]
I am not from the train cargo industry, so it is hard to provide any valuable/meaningful input.
Right now my main concern is vehicle transport, but what comes to my mind is also a characteristic of a given shuttle train - is it permitted for trucks, and what are limitations? I guess costs/schedule times/speed are out of the scope.
